modules/picard/liftovervcf/meta.yml

```diff
@@ -0,0 +1,52 @@
+name: picard_liftovervcf
+description: convert between genome builds
+keywords:
+  - liftOver
+  - picard
+tools:
+  - picard:
+      description: Move annotations from one assembly to another
+      homepage: https://gatk.broadinstitute.org/hc/en-us/articles/360037060932-LiftoverVcf-Picard
+      documentation: https://gatk.broadinstitute.org/hc/en-us/articles/360037060932-LiftoverVcf-Picard
+      tool_dev_url: https://github.com/broadinstitute/picard
+      doi: ""
+      licence: ["MIT"]
+
+input:
+  - meta:
+      type: map
+      description: Groovy Map containing sample information
+        e.g. [ id:'test', single_end:false ]
+  - input_vcf:
+      type: file
+      description: VCF file
+      pattern: "*.{vcf,vcf.gz}"
+  - chain:
+      type: file
+  - fasta:
+      type: file
+      description: fasta file
+  - dict:
+      type: file
+      description: dictionary for fasta file
+
+output:
+  - meta:
+      type: map
+      description: Groovy Map containing sample information
+        e.g. [ id:'test', single_end:false ]
+  - versions:
+      type: file
+      description: File containing software versions
+      pattern: "versions.yml"
+  - vcf:
+      type: file
+      description: VCF file containing successfully lifted variants
+      pattern: "*.{vcf,vcf.gz}"
+  - vcf_unlifted:
+      type: file
+      description: VCF file containing unsuccessfully lifted variants
```

Comment:
Alternative description: "VCF files containing variants that couldn't be lifted."

Should the file pattern be `".{unlifted.vcf,unlifted.vcf.gz}"` ? (if yes, do the same for the other VCF output)

Reply:
Thanks for the suggestion, I have added it in the latest commit